src/Inventory/Inventory.php

```diff
@@ -494,7 +494,7 @@ public static function getMenuContent()
             ];
         }
 
-        if (Session::haveRight(Lockedfield::$rightname, READ)) {
+        if (Session::haveRight(Lockedfield::$rightname, CREATE)) {
```

Comment:
Not sure, but I think that only the `Add global lock` link should be removed if user has not the CREATE right.

Reply:
Seems there is no READ right for LocakedFields, so it seems correct to rely on CREATE for the whole

Reply:
Indeed. Should a READ right be added ?

Reply:
No ? Is there a scenario where a tech user needs to see global locks **configuration** but not be able to update (create/delete) them

Reply:
I don't think so. Proposed change seems correct to me

Reply:
`$menu['options']['lockedfield']` entry have to be present, even if user has not the `Session::haveRight(Lockedfield::$rightname, CREATE)``right, in order to have the corresponding entry in breadcrumbs.

Reply:
> $menu['options']['lockedfield'] entry have to be present

I'm lost here.
I'm not sure the entry should be present.
If the user doesn't have right to alter locks, the entry should probably not be here at all.